services/ui/src/components/Project/index.js

```diff
@@ -68,6 +72,12 @@ class Project extends React.Component {
               <div className='field'>{this.props.project.pullrequests}</div>
             </div>
           </div>
+          <div className='field-wrapper envlimit'>
+            <div>
+              <label>Non-Production environments in use</label>
+              <div className='field'>{this.state.developEnvironmentCount} of {this.props.project.developmentEnvironmentsLimit} </div>
```

Comment:
Is `developmentEnvironmentsLimit` always a number? It looks like it can be set to null to mean unlimited? Think it would be nice to display "x of unlimited" or "x of ∞" if that's the case.

Reply:
That's a good point.  New projects are created with a default (5), but it is nullable so that's something I should consider.

Reply:
I've updated this to safely tolerate null.